proto/protovalidate/buf/validate/validate.proto

```diff
@@ -4288,7 +4288,7 @@ message TimestampRules {
     // ```
     bool lt_now = 7 [(priv.field).cel = {
       id: "timestamp.lt_now"
-      expression: "this > now ? 'value must be less than now' : ''"
+      expression: "rules.lt_now ? (this > now ? 'value must be less than now' : '') : ''"
```

Comment:
Hate to be a pain but those nested ternaries are kind of unwieldy. WDYT about something like this? I'm not sure if it's really better, though...

```suggestion
      expression: "(!rules.lt_now || this > now) ? 'value must be less than now' : ''"
```

Reply:
I think that's actually nicer! Fixed in [1b0942a](https://github.com/bufbuild/protovalidate/pull/252/commits/1b0942a00523613ac276782431b4a74372b2f304)

Reply:
Thanks!